src/Npgsql/NpgsqlBinaryImporter.cs

```diff
@@ -284,9 +284,10 @@ public void WriteRow(params object[] values)
         /// <summary>
         /// Completes the import operation. The writer is unusable after this operation.
         /// </summary>
-        public void Complete()
+        public int Complete()
         {
             CheckReady();
+            var copyRows = 0; // number of rows copied on success. See "Outputs" section of https://www.postgresql.org/docs/current/static/sql-copy.html
```

Comment:
Any reason to declare this here rather than just returning the value directly below?

Reply:
I'll double check this. At some point a default of 0 should be set.